torch/optim/lr_scheduler.py

```diff
@@ -187,8 +185,9 @@ def __exit__(self, type, value, traceback):
 
 
 class LambdaLR(LRScheduler):
-    """Sets the learning rate of each parameter group to the initial lr
-    times a given function. When last_epoch=-1, sets initial lr as lr.
+    """Sets the learning rate of each parameter group to the initial lr times a given function.
```

Comment:
```suggestion
    """Set the learning rate of each parameter group to the initial lr times a given function.
```
Should this be singularized following the pattern everywhere else?

Reply:
Classes don't necessarily have to start with an imperative verb.
Therefore, I didn't modify the class docstrings to make them imperative.